webrender/src/renderer.rs

```diff
@@ -1481,6 +1517,93 @@ impl Renderer {
                 ResultMsg::RefreshShader(path) => {
                     self.pending_shader_updates.push(path);
                 }
+                ResultMsg::DebugCommand(command) => {
+                    self.handle_debug_command(command);
+                }
+            }
+        }
+    }
+
+    #[cfg(not(feature = "debugger"))]
+    fn update_debug_server(&self) {
+    }
```

Comment:
perhaps, would be cleaner to do `let _ = &self.debug_server` than having underscore in `_debug_server`